File: PHP/Program/Kategori.php
<?php
class Kategori {
    // atribut private
    private $idKategori;
    private $namaKategori;
    private $deskripsi;

    // constructor
    public function __construct($idKategori = "", $namaKategori = "", $deskripsi = "") {
        $this->idKategori = $idKategori;
        $this->namaKategori = $namaKategori;
        $this->deskripsi = $deskripsi;
    }

    // getters
    public function getIdKategori() {
        return $this->idKategori;
    }
    public function getNamaKategori() {
        return $this->namaKategori;
    }
    public function getDeskripsi() {
        return $this->deskripsi;
    }

    // setters
    public function setIdKategori($idKategori) {
        $this->idKategori = $idKategori;
    }
    public function setNamaKategori($namaKategori) {
        $this->namaKategori = $namaKategori;
    }
    public function setDeskripsi($deskripsi) {
        $this->deskripsi = $deskripsi;
    }
}

File: Java/Program/Kategori.java
package Java.Program;

public class Kategori {
    private String idKategori;
    private String namaKategori;
    private String deskripsi;

    public Kategori(String idKategori, String namaKategori, String deskripsi) {
        this.idKategori = idKategori;
        this.namaKategori = namaKategori;
        this.deskripsi = deskripsi;
    }

    // Getters
    public String getIDKategori() { return idKategori; }
    public String getNamaKategori() { return namaKategori; }
    public String getDeskripsi() { return deskripsi; }

    // Setters
    public void setIDKategori(String idKategori) { this.idKategori = idKategori; }
    public void setNamaKategori(String namaKategori) { this.namaKategori = namaKategori; }
    public void setDeskripsi(String deskripsi) { this.deskripsi = deskripsi; }
}
